stylix/icons.nix

```diff
@@ -1,6 +1,20 @@
 { lib, ... }:
 {
-  options.stylix.iconTheme = {
+  imports = [
+    (lib.mkRenamedOptionModuleWith {
+      from = [
+        "stylix"
+        "iconTheme"
+      ];
+      sinceRelease = 2511;
+      to = [
+        "stylix"
+        "icons"
+      ];
+    })
+  ];
```

Comment:
Just in case anyone else comes across error from rename and was using priorities. 

```bash
  error: A definition for option `home-manager.users.khaneliman.stylix.icons.enable' is not of type `boolean'. Definition values:
┃        - In `/nix/store/yd0pp118w6l7kjqcncd9nlrxsd0a3gi6-source/stylix/icons.nix':
┃            {
┃              _type = "override";
┃              content = true;
┃              priority = 1000;
┃            }
```